### ISSUE TITLE
A11y_VSCodespaceBenchmark_Defaultsettingfornewcodespaces_Showpricinginfo_Keyboard : The tooltip for controls such as "Create New Copespace", "Select Plan" and "Refresh" is not visible using keyboard.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode#106110]
"[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Environment Details:​
Application Name: VS Codespace
Application Version: 1.48.0-insider
Commit: 0a7364f00514c46c9caceece15e1f82f82e3712f
Date: 2020-07-22T05:43:52.797Z
Electron: 9.1.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
Operating System: MAC 10.15.6
Screen Reader: Voiceover
Standard Violated: WCAG 1.1.1, WCAG 2.1.1

### Repro Steps:​
1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Online"
3. Click on "Visual Studio Online" and navigate till "Remote Explorer" ,click on it.
4. Navigate the controls such as "Create New Copespace", "Select Plan" and "Refresh" using keyboard.
5. Observe if the tooltip that is available for the mouse users is available for keyboard users or not.

### Actual:
When keyboard users navigate to the controls such as "Create New Copespace", "Select Plan" and "Refresh" using the 'Tab' key the tooltip content is not available for keyboard users which is visible when mouse hovered over these controls which is incorrect.

### Expected: ​
The tooltip for controls such as "Create New Copespace", "Select Plan" and "Refresh" should be available for the keyboard users also when they navigate to these controls.

### ​User Impact:
If the tooltip content available for the controls on the screen is not available fo the keyboard only users then they will miss out on the information that is available for the mouse users.

### Attachment:
[1166171_A11y_VSCodespaceBenchmark_Defaultsettingfornewcodespaces_Showpricinginfo_Keyboard The tooltip for controls such as "Create New Copespace", "Select Plan" and "Refresh" is not visible using keyboard.mov.zip](https://github.com/microsoft/vscode/files/5174329/1166171_A11y_VSCodespaceBenchmark_Defaultsettingfornewcodespaces_Showpricinginfo_Keyboard.The.tooltip.for.controls.such.as.Create.New.Copespace.Select.Plan.and.Refresh.is.not.visible.using.keyboard.mov.zip)

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!

**Comment by MayuriNaik** [comment]
@isidorn We have already logged this bug for VS Codespace and they have redirected us to log an external bug for VS Code. So we logged this bug here.
